java/java-ranger-regression/nanoxml_eqchk/prop1/Main.java

```diff
@@ -0,0 +1,30 @@
+import java.util.ArrayList;
+import org.sosy_lab.sv_benchmarks.Verifier;
+
+public class Main {
+
+    public static void main(String[] args) {
+        char c0 = Verifier.nondetChar();
+        char c1 = Verifier.nondetChar();
+        char c2 = Verifier.nondetChar();
+        char c3 = Verifier.nondetChar();
+        char c4 = Verifier.nondetChar();
+        char c5 = Verifier.nondetChar();
+        char c6 = Verifier.nondetChar();
+        char c7 = Verifier.nondetChar();
+        char c8 = Verifier.nondetChar();
+        DumpXML d = new DumpXML();
+        int out1 = d.mainProcess(c0, c1, c2, c3, c4, c5, c6, c7, c8);
+        d.numIdentifiers = 0; // commenting this line out causes the equivalence check to fail
```

Comment:
As `numIdentifiers` is a static variable, I think you need `DumpXML.numIdentifiers = 0;`. Or do I miss something in the Java language here?